### DISCUSSION TITLE
Showing Keyword Data Sent and also is there a way to filter by day etc

### DISCUSSION THREAD

**Discussion by ppapasav** [allinurl/goaccess#2391] [Q&A]
First, great tool - Thanks for taking the time to share your creation with us

Two questions

First: I see under traffic, referring URL - The Logs do capture Keywords used (in advertising campaigns), but I don't see it reported anywhere in goaccess (i could have just missed it)

Second
Is there a way to show a report for a specific day as opposed to showing all log traffic

**Comment by allinurl** [comment]
Thanks. Currently it attempts to capture Google query keywords. However, since Google stopped passing them on the referral, I plan to replace that panel for UTM_* campaign data. 

As far as filtering, #117 will address this in a nice a way. I’m working on this as we speak. [Stay tuned](https://twitter.com/goaccess).
